pkg/store/change/device/utils_test.go

```diff
@@ -0,0 +1,409 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package device
+
+import (
+	"encoding/base64"
+	"github.com/golang/mock/gomock"
+	"github.com/onosproject/onos-config/pkg/store/change"
+	mockstore "github.com/onosproject/onos-config/pkg/test/mocks/store"
+	devicechange "github.com/onosproject/onos-config/pkg/types/change/device"
+	types "github.com/onosproject/onos-config/pkg/types/change/device"
+	devicepb "github.com/onosproject/onos-topo/pkg/northbound/device"
```

Comment:
We have started calling this `devicetopo` to make it clear it's from `topo` because `devicepb` is so ambiguous